ext/featurens/redhatrelease/redhatrelease.go

```diff
@@ -32,8 +32,9 @@ import (
 var (
 	amazonReleaseRegexp = regexp.MustCompile(`(?P<os>Amazon) (Linux release|Linux AMI release) (?P<version>[\d]+\.[\d]+|[\d]+)`)
 	oracleReleaseRegexp = regexp.MustCompile(`(?P<os>Oracle) (Linux Server release) (?P<version>[\d]+)`)
-	centosReleaseRegexp = regexp.MustCompile(`(?P<os>[^\s]*) (Linux release|release) (?P<version>[\d]+)`)
```

Comment:
This change limits the strings that we are matching - would be interested in opinion of @RTann on this.

Reply:
I'm thinking let's revert this to ensure that continues to work as before